docs/src/tutorials/exporting.md

```diff
@@ -1,51 +1,176 @@
 ---
 title: "Exporting data"
-description: |
-  Platform.sh aims to be a great host, but we never want to lock you in to our service. Your code and your data belong to you, and you should always be able to download your site's data for local development, backup, or to "take your data elsewhere".
+description: See how to export your code, files and service's data.
 ---
 
-{{% description %}}
+Platform.sh never wants to lock you in to our service.
+Your code and your data belong to you,
+and you should always be able to download your site's data for local development,
+backup, or to "take your data elsewhere".
+## Before you begin
 
-## Downloading code
+You need:
```

Comment:
😍 I like that you made the list items as short as possible. I should strive to do that as well.